cluster/addons/fluentd-gcp/README.md

```diff
@@ -0,0 +1,11 @@
+# Stackdriver Logging Agent
+==============
+
+Stackdriver Logging Agent is a DaemonSet which spawns a pod on each node
+that reads logs, generated by kubelet, container engine and containers
```

Comment:
what is container engine? is kubelet a part of container engine? is docker a part of container engine?